packages/context/test/unit/binding.unit.ts

```diff
@@ -52,6 +52,28 @@ describe('Binding', () => {
       expect(binding.tags.has('t1')).to.be.true();
       expect(binding.tags.has('t2')).to.be.true();
     });
+
+    it('tags the binding with rest args', () => {
+      binding.tag('t1', ['t2', 't3'], 't4');
+      expect(binding.tags.has('t1')).to.be.true();
+      expect(binding.tags.has('t2')).to.be.true();
+      expect(binding.tags.has('t3')).to.be.true();
+      expect(binding.tags.has('t4')).to.be.true();
+    });
+  });
+
+  describe('tagValue', () => {
+    it('tags the binding with value', () => {
+      binding.tagValue('t1', 'val1');
+      expect(binding.tags.has('t1')).to.be.true();
+      expect(binding.tagValues.get('t1')).to.equal('val1');
+    });
+
+    it('tags the binding with a name', () => {
+      binding.tag('t1');
+      expect(binding.tags.has('t1')).to.be.true();
+      expect(binding.tagValues.get('t1')).to.equal(true);
```

Comment:
Ditto.

```ts
expect(/*convert tags to an object*/).to.deepEqual({t1: true});
```

Please improve the test name to mention what value is being used, e.g.

```ts
it('sets the tag value to true when tagging by a name only');
```